packages/shared/src/components/MainFeedLayout.tsx

```diff
@@ -14,7 +14,8 @@ import { LoggedUser } from '../lib/user';
 import OnboardingContext from '../contexts/OnboardingContext';
 import MagnifyingIcon from '../../icons/magnifying.svg';
 import { Dropdown, DropdownProps } from './fields/Dropdown';
-import { Button, ButtonProps } from './buttons/Button';
+// eslint-disable-next-line import/no-named-as-default
```

Comment:
Are you sure this is needed? I wouldn't want to add this line wherever we import button